detd/devices/intel_i210.py

```diff
@@ -35,4 +46,49 @@ def __init__(self, pci_id):
 
         logger.info(f"Initializing {__class__.__name__}")
 
-        raise NotImplementedError("Handler class for i210 Ethernet controller not yet implemented")
+        super().__init__(IntelI210.NUM_TX_QUEUES, IntelI210.NUM_RX_QUEUES)
+
+        self.features['rxvlan'] = 'off'
+
+        # self.num_tx_ring_entries and self.num_rx_ring_entries
+        # Provides the number of ring entries for Tx and Rx rings.
+        # Currently, the code just passes the value to ethtool's --set-ring.
+        self.num_tx_ring_entries = 1024
+        self.num_rx_ring_entries = 1024
+
+    def get_rate(self, interface):
+
+        # Without a small delay, the program flow will call ethtool twice too
+        # fast, causing it to return "Unknown!" speed
+        time.sleep(1)
+
+        return self.systeminfo.get_rate(interface)
+
+
+    def get_base_time_multiple(self):
+        return -1
+
+
+    def supports_schedule(self, schedule):
+
+        return True
+
+    def device_hints(self):
+        # Check the capabilities and set the Hints
+
+        preemption = False
+        launch_time_control = False
+        TxSelectionOffload = False
+        datapath = DataPath.AF_PACKET
+        txcapability = TxCapability.none
+    
+        if Capability.Qbv in self.capabilities:
+            txcapability = TxCapability.Qbv
+        if Capability.Qbu in self.capabilities and not Capability.Qbv in self.capabilities:
```

Comment:
By **default**, if Qbv HW is not supported just set offload, preemption and LTC as False. No need to check further. That would be a sane default.